src/osd/OSD.cc

```diff
@@ -3667,6 +3667,23 @@ float OSD::get_osd_recovery_sleep()
     return cct->_conf->osd_recovery_sleep_hdd;
 }
 
+float OSD::get_osd_recovery_sleep_degraded() {
+  float osd_recovery_sleep_degraded = 
+    cct->_conf.get_val<double>("osd_recovery_sleep_degraded");
+  if (osd_recovery_sleep_degraded > 0) {
+    return osd_recovery_sleep_degraded;
+  }
+  if (!store_is_rotational && !journal_is_rotational) {
+    return cct->_conf.get_val<double>("osd_recovery_sleep_degraded_ssd");
+  }
+  else if (store_is_rotational && !journal_is_rotational) {
```

Comment:
We tend to put 'else if' on the same line after ending '}' through the Ceph code base. IMO better to follow the same rule here